ourrevolution/settings.py

```diff
@@ -279,6 +279,10 @@
 NOMINATIONS_QUESTION_ISSUES_ID = 24
 
 ADD_GROUP_URL = '/groups/new/'
+ELECTORAL_COORDINATOR_EMAIL = str(os.environ.get(
```

Comment:
why do we need `str` here? we don't use it elsewhere.

Reply:
To allow `.env` to be 'email@example.com' or email@example.com and still work.